sklearn/ensemble/_hist_gradient_boosting/tests/test_gradient_boosting.py

```diff
@@ -744,3 +744,38 @@ def test_single_node_trees(Est):
 def test_custom_loss(Est, loss, X, y):
     est = Est(loss=loss, max_iter=20)
     est.fit(X, y)
+
+
+@pytest.mark.parametrize('HistGradientBoosting, X, y', [
+    (HistGradientBoostingClassifier, *_make_dumb_dataset(1200)),
+    (HistGradientBoostingRegressor, *_make_dumb_dataset(1200))
```

Comment:
I'd suggest using

```py
    (HistGradientBoostingClassifier, X_classification, y_classification),
    (HistGradientBoostingRegressor, X_regression, y_regression)
```

instead. Also, we'd need to test the multiclass case. You can declare `X_multi_classification` and `y_multi_classification` at the top of the file

Reply:
I use 3-class for multi-class.